CONTRIBUTING.md

```diff
@@ -141,7 +141,7 @@ Ensure you use version 6.4.0 to prevent conflicts. Check the appropriate release
 
 ## Coding style
 
-We follow [google coding style](http://google.github.io/styleguide/).
+We follow the [Google coding style](http://google.github.io/styleguide/).
```

Comment:
```suggestion
We follow the [Google style guide](http://google.github.io/styleguide/).
```

Reply:
Deprecated later when I get rid of it when I reorganize sections, but a good idea to fix in this commit.